packages/frontend/src/components/MkEmojiPicker.vue

```diff
@@ -5,7 +5,20 @@ SPDX-License-Identifier: AGPL-3.0-only
 
 <template>
 <div class="omfetrab" :class="['s' + size, 'w' + width, 'h' + height, { asDrawer, asWindow }]" :style="{ maxHeight: maxHeight ? maxHeight + 'px' : undefined }">
-	<input ref="searchEl" :value="q" class="search" data-prevent-emoji-insert :class="{ filled: q != null && q != '' }" :placeholder="i18n.ts.search" type="search" autocapitalize="off" @input="input()" @paste.stop="paste" @keydown.stop.prevent.enter="onEnter">
+	<input
+		ref="searchEl"
+		:value="q"
+		class="search"
+		data-prevent-emoji-insert
+		:class="{ filled: q != null && q != '' }"
+		:placeholder="i18n.ts.search"
+		type="search"
+		autocapitalize="off"
+		@input="input()"
+		@paste.stop="paste"
+		@keydown.stop.prevent.enter="onEnter"
+		@keydown.stop.prevent.esc="emit('esc')"
```

Comment:
個人的にonKeydownでまとめたい感じがする
EscapeキーにもisComposingなどの判定をさせたい

```suggestion
		@keydown="onKeydown"
```

```ts
function onKeydown(ev: KeyboardEvent) {
  if (ev.isComposing || ev.key === 'Process' || ev.keyCode === 229) return;
	if (ev.key === 'Enter') {
		ev.preventDefault();
		ev.stopPropagation();
		done();
	}
	if (ev.key === 'Escape') {
		ev.preventDefault();
		ev.stopPropagation();
		emit('esc');
	}
}
```

Reply:
`@keydown.stop="onKeydown"` とするなら if内の `ev.stopPropagation();` はいらないかも

Reply:
foxed